apps/mobile/src/navigation/useScrollableHeader.tsx

```diff
@@ -28,7 +28,12 @@ export const useScrollableHeader = ({ children, alwaysVisible, scrollYThreshold
   useEffect(() => {
     navigation.setOptions({
       headerTitle: () => (
-        <Animated.View style={[{ flexDirection: 'row', alignItems: 'center' }, animatedHeaderStyle]}>
+        <Animated.View
+          style={[
+            { flex: 1, justifyContent: 'center', flexDirection: 'row', alignItems: 'center' },
+            animatedHeaderStyle,
+          ]}
+        >
```

Comment:
did you test other views that use this hook. Any noticeable change? Why are we adding flex 1 and justfiy content center?